Sources/StreamVideo/Call.swift

```diff
@@ -81,6 +82,10 @@ public class Call: @unchecked Sendable, WSEventsSubscriber {
         }
     }
 
+    deinit {
```

Comment:
Not convinced if this is the best place to handle this. The `Call` object doesn't setup, nor know anything about the `AVAudioSession`, so this feels a bit hacky.
When a call is ended, a `CallNotification.callEnded` is sent, so maybe the `MicrophoneChecker` can react to that? Or if you have any other idea, that's also fine.